src/apm.ts

```diff
@@ -1,9 +1,22 @@
 import { EventEmitter } from 'events';
 import type { Callback } from './utils';
+import { Connection } from './cmap/connection';
```

Comment:
shouldn't this be an `import type`?

Reply:
No I'm using the static properties defined on it so I need the value.